packages/commons/src/events/test.utils.ts

```diff
@@ -226,7 +226,7 @@ export function generateActionDocument({
   defaults?: Partial<ActionDocument>
 }): ActionDocument {
   const actionBase = {
-    createdAt: new Date().toISOString(),
+    createdAt: new Date(Date.now() - 500).toISOString(),
```

Comment:
Not the best fix. We need it to be so the creation timestamps for events, actions and drafts make logical sense in storybook tests